apps/emqx/src/emqx_trace/emqx_trace.erl

```diff
@@ -111,7 +111,10 @@ rendered_action_template(ActionID, RenderResult) ->
         _ ->
             ok
     end,
-    TraceResult.
+    TraceResult;
+rendered_action_template(_ActionID, _RenderResult) ->
+    %% We do nothing if we don't get a valid Action ID
```

Comment:
under which condition do we get an invalid action id ?

Reply:
ok, maybe add as a comment.

Reply:
Comment added.